user_technical/how_to_guides/import-data.md

```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
 ```
 
 Next time you run Memgraph, the dataset will be loaded.
+
+### Importing Cypher Queries {#import-cypher}
+
+When Memgraph is running, cypher queries are imported by running `mg_client` in
+non-interactive mode. The user can import queries saved in e.g. `queries.txt`
+by issuing the following shell command:
+
+`mg_client < queries.txt`
+
+If you installed Memgraph using Docker, you will need to run the client using
+the following command:
+
+`docker run -i --entrypoint=mg_client memgraph --host HOST < queries.txt`
+
+Remember to replace `HOST` with valid IP of the container(see
```

Comment:
space between container and open parenthesis. This seems like you're calling the container function :)